### ISSUE TITLE
Add bower support

### ISSUE THREAD

**Issue by Siliconrob** [zippyui/react-datagrid#74]
Add bower support and listing for this project.

**Comment by Siliconrob** [comment]
The files under the dist folder work well enough, so if you added a bower entry for the project it would be ready.

**Comment by radubrehar** [comment]
Yes, we have used `UMD` export in the `dist` folder. Will add a bower.json file and register with bower registry. thanks

**Comment by monarchjhaveri** [comment]
Would also like this functionality. Thanks.

**Comment by denis-itskovich** [comment]
+1

**Comment by Nikser** [comment]
+1
